packages/react-core/src/helpers/Popper/Popper.tsx

```diff
@@ -54,9 +54,11 @@ export interface PopperProps {
    * The reference element to which the Popover is relatively placed to.
    * Use either trigger or reference, not both.
    */
-  reference?: HTMLElement | (() => HTMLElement) | React.RefObject<any>;
+  triggerRef?: HTMLElement | (() => HTMLElement) | React.RefObject<any>;
```

Comment:
Oh actually I can remove that line entirely, as the Popper allows both now props now. Will do another pass to update other `reference` references

Reply:
What do you think about the following prop descriptions?

"Trigger element to which the popper is relatively placed to."

"A reference to the trigger element that can be passed instead of or along with the trigger prop. When passed along with the trigger prop, the div element that wraps the trigger will be removed."

Would also be worth adding onto the popperRef description to mention that "Passing this prop will remove the wrapper div element from the popper."